lib/device_api/android/plugins/battery.rb

```diff
@@ -0,0 +1,22 @@
+module DeviceAPI
+  module Android
+    module Plugin
+      class Battery
+        attr_accessor :current_temp, :max_temp, :max_current, :voltage, :level, :health, :status
```

Comment:
Just a point of style (which you may decide to ignore). I would favour using attr_reader instead of attr_accessor and then setting these in the initialise method using @current_temp, etc, as you do not really want these values to be set from outside.